lexers/g/groff.go

```diff
@@ -0,0 +1,42 @@
+package g
+
+import (
+	"regexp"
+	"unicode"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var groffAlphanumericRe = regexp.MustCompile(`^[a-zA-Z0-9.]+$`)
+
+// Groff lexer.
+var Groff = internal.Register(MustNewLexer(
+	&Config{
+		Name:      "Groff",
+		Aliases:   []string{"groff", "nroff", "man"},
+		Filenames: []string{"*.[1234567]", "*.man"},
+		MimeTypes: []string{"application/x-troff", "text/troff"},
+	},
+	Rules{
+		"root": {},
+	},
+).SetAnalyser(func(text string) float32 {
+	if text[:1] != "." {
+		return 0
+	}
+
+	if text[:3] == ".\\" {
```

Comment:
In python, the pattern is `'.\\"'`. I guess this would equal `` `.\"` ``. Do you know more about this pattern in Groff? You didn't add a test for it, but can we somehow track down an example for it?

Reply:
it's a comment for `groff` and has been added. tks.